scene/animation/animation_node_state_machine.cpp

```diff
@@ -349,7 +351,7 @@ double AnimationNodeStateMachinePlayback::process(AnimationNodeStateMachine *p_s
 		if (p_state_machine->start_node != StringName() && p_seek && p_time == 0) {
 			current = p_state_machine->start_node;
 		}
-
+		emit_signal(SceneStringNames::get_singleton()->state_enter, current);
```

Comment:
No need to use SceneStringNames any more, you can do SNAME("state_enter") which is as efficient.